build/NuSpecs/WindowsAppSDK-Nuget-Native.C.props

```diff
@@ -28,9 +28,12 @@
         %(AdditionalDependencies);
       </AdditionalDependencies>
     </Link>
-    <PostBuildEvent>
-      <Command>xcopy.exe /y "$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(_WindowsAppSDKFoundationPlatform)\native\Microsoft.WindowsAppRuntime.Bootstrap.dll" "$(OutDir)"</Command>
-    </PostBuildEvent>
   </ItemDefinitionGroup>
 
+  <Target Name="CopyMicrosoftWindowsAppRuntimeBootstrapdllToOutDir" Condition="'$(AppxPackage)' != 'true'" AfterTargets="Build">
```

Comment:
Why is this conditioned on "not an appx package"? I guess because generally appx uses the FWP model of binding and not bootstrap?

Reply:
I don't know, I just maintained the condition from where it was previously. Probably a question for whoever added this -- @DrusTheAxe added in https://github.com/microsoft/WindowsAppSDK/pull/827.

I also see that Howard tried to make a change to Copy in #2734 and put it back on #2758. Hopefully someone more knowledgeable about these can help verify this change doesn't re-break any of those scenarios. I suspect the change in #2734 didn't work because it wasn't invoking the task in a Target.

Reply:
OK. I'm generally supportive of the change. If you wanted a slam-dunk no-questions-asked change, just add the `/d` parameter to xcopy, which only copies if the source is newer than the target.

Reply:
>Why is this conditioned on "not an appx package"?
Because Bootstrapper is only needed when WinAppSDK MSIX packages via WinAppSDK DynamicDependencies, AND WinAppSDK DynDep only supports unpackaged apps.

But isn't this necessary-but-insufficient -- the condition should be "I an not a packaged app AND i am not using WinAppSDKSelfContained"?

```
Condition="('$(AppxPackage)' != 'true') and ('$(WindowsAppSDKSelfContained)' != 'true')"
```

?